### PR TITLE
(PE-17212) Update module to properly manage dujour as a tk service

### PR DESCRIPTION
**Pull request by briancain**
Now that dujour runs as a tk service, this commit updates the puppet
module with the new way to manage its config. It also adds some unit
tests for the dujour config file.

### CODE REVIEW

**Comment by briancain** [comment]
FYI this should be okay to merge out of band of deployment. Ops is using this module but it's properly pinned to the bd471ec commit so we shouldn't mess anything up my merging this.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ The Dujour module provides a quick way to get started using Dujour, a version-ch
 
 For more information about Dujour [please see the official Dujour documentation.](https://github.com/puppetlabs/dujour)
 
+Tests
+------
+
+To run the dujour puppet module tests, simply type the following below:
+
+```
+$ rake spec
```
Comment (smcclellan) [inline README.md]: `bundle exec rake spec` might be better here, since the user's default ruby may differ from what bundler uses.

Comment (smcclellan) [inline README.md]: Probably also add a line here to run `bundle install` first.

Comment (briancain) [inline README.md]: 👍  I forget I have some magic in my shell to pretty much do that for me...I'll update the docs!

**Comment by smcclellan** [comment]
Looks good to me!

**Pull request merged**